steward-auto-merge.yaml

```diff
@@ -0,0 +1,17 @@
+name: Dependabot auto-merge
+on: pull_request
+
+permissions:
+  contents: write
+  pull-requests: write
+
+jobs:
+  dependabot:
+    runs-on: ubuntu-latest
+    if: ${{ github.actor == 'scala-steward-for-moia[bot]' || github.actor == 'houcros' }}
```

Comment:
The last part of the condition is for testing. Will delete it in a follow-up PR, once testing is done.